tests/runtest.sh

```diff
@@ -231,18 +231,20 @@ run_corefx_tests()
         # Check if we're running under Linux
         Linux)
             TESTS_REMOTE_URL=$(<${CoreRT_TestRoot}/CoreFXTestListURL_Linux.txt)
+            TEST_LIST_JSON=${CoreRT_TestRoot}/TopN.CoreFX.Linux.issues.json
```

Comment:
I think we can have one set of issues for all Unixes. It is very rare for runtime issues to show up on a single Unix flavor only (and we can disabled the test for all Unixes when that happens).